documentation/src/main/asciidoc/topics/con_file_based_cache_stores.adoc

```diff
@@ -4,6 +4,12 @@
 File-based cache stores provide persistent storage on the local host filesystem where {brandname} is running.
 For clustered caches, file-based cache stores are unique to each {brandname} node.
 
+[IMPORTANT]
+====
+To retain data during clustered cache restarts, use a shared store. 
+See, link:https://access.redhat.com/solutions/6960765[Adding persistence in a cache in RHDG 8 in OCP 4] ({RedHat} Customer Portal). 
+====
+
```

Comment:
The following KCS (https://access.redhat.com/solutions/1249463) will describe without OCP environment, might be worth to have both

Reply:
KCS aren't upstream friendly as they're behind a paywall. All required information should be covered in our docs.

Reply:
but the KCS might help to extract the information :)